.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,7 +1,7 @@
-Please do not send pull requests to the BoringSSL repository.
+Thank you for contributing to OQS-BoringSSL! Before submitting your pull request, please ensure the following:
 
-We do, however, take contributions gladly.
+1. Your changes include new tests or modifications to existing tests as needed.
 
-See https://boringssl.googlesource.com/boringssl/+/master/CONTRIBUTING.md
+2. You have tested the changes in a clean build environment.
 
-Thanks!
+3. Your changes do not break the build for projects that depend on OQS-BoringSSL.
```

Comment:
I like that wording. For the "uninitiated", would it be sensible to create a pointer (at least) to oqs-demos (curl/nginx-quic), maybe even a script a new contributor could run so any PR doesn't break something there? Or even a downstream-test-build trigger for CI?

Reply:
updated

Reply:
Please do with hyperlinks to oqs-demos. Just using the terms here may not be helpful for the "uninitiated" (not knowing about where/what's in oqs-demos). Apologies for being a pain-in-the-butt, but (one-time) clarity removes (many-time) pointless questions.

Reply:
@baentsch Done